dapp/src/services/balances.service.js

```diff
@@ -0,0 +1,105 @@
+import { ethers } from 'ethers'
+import { displayCurrency } from 'utils/math'
+import { isProduction } from 'constants/env'
+
+let jsonCallId = 1
+
+export default class BalancesService {
+  async fetchBalances(account, contracts) {
+    if (isProduction) {
+      return this.fetchBalancesForProduction(account, contracts)
+    }
+    return this.fetchBalancesForDevelopment(account, contracts)
+  }
+
+  async fetchBalancesForProduction(account, contracts) {
```

Comment:
I like how you use more verbose names for functions. Really makes the code more readable, which is a benefit for everyone going through it.